docs/releases/3.1.1.txt

```diff
@@ -25,3 +25,6 @@ Bugfixes
   :meth:`~django.db.models.query.QuerySet.select_for_update()`. When using
   related fields pointing to a proxy model in the ``of`` argument, the
   corresponding model was not locked (:ticket:`31866`).
+
+* Fixed type inspection of deprecated keyword arguments in
+  ``TemplateView.get_context_data()`` (:ticket:`13297`).
```

Comment:
I think it's better to describe a real crash because that should make it easier for folks to identify that this is the issue they're facing.

Reply:
Agree